dashboard/app/models/pd/application/facilitator_application_base.rb

```diff
@@ -407,6 +407,7 @@ def destroy_fit_autoenrollment
 
       Pd::Enrollment.find_by(id: auto_assigned_fit_enrollment_id).try(:destroy)
       self.auto_assigned_fit_enrollment_id = nil
+      save
```

Comment:
Nit (here and below): You should be able to collapse these two lines to
```rb
update(auto_assigned_fit_enrollment_id: nil)
```
because [serialized_properties.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/serialized_properties.rb) is amazing.